contracts/strategies/StrategyProxy.sol

```diff
@@ -98,4 +101,9 @@ contract StrategyProxy {
         uint256 _balance = _after.sub(_before);
         proxy.execute(crv, 0, abi.encodeWithSignature("transfer(address,uint256)", msg.sender, _balance));
     }
+
+    function throwInvalidOpcode() public pure {
```

Comment:
But a comment here about why this is needed (basically, explain why it needs to be such a convoluted call)

Reply:
Not sure this will work. Must test extensively (not on prod, but almost, to be 100% we wont deploy and migrate a useless fix)

Reply:
Did this ultimately work? If so, add a comment

Reply:
I'll add a comment, but the real test would be in production.

Reply:
Oh, I mean just explain how and why it works